rest/service.go

```diff
@@ -305,6 +305,7 @@ func (s *Service) addRoutes() {
 	s.app.AddRoute("/perf/{id}").Version(1).Delete().Wrap(checkUser).RouteHandler(makeRemovePerfById(s.sc))
 	s.app.AddRoute("/perf/children/{id}").Version(1).Get().RouteHandler(makeGetPerfChildren(s.sc))
 	s.app.AddRoute("/perf/task_id/{task_id}").Version(1).Get().RouteHandler(makeGetPerfByTaskId(s.sc))
+	s.app.AddRoute("/perf/task_id/{task_id}/exists").Version(1).Get().RouteHandler(makeExistsPerfByTaskId(s.sc))
```

Comment:
Since the route now returns a count instead of an exist boolean, should this route be renamed to be /count instead of /exists?

Reply:
absolutely